.github/workflows/metallb_e2e.yml

```diff
@@ -43,9 +43,9 @@ jobs:
       - name: Checkout MetalLB
         uses: actions/checkout@v2
         with:
-          repository: metallb/metallb
+          repository: fedepaol/metallb
```

Comment:
is that temp for testing? maybe worth adding a comment so we won't forget before merging.

Reply:
yes @sabinaaledort , this is just for testing purpose, will update it once this [PR](https://github.com/metallb/metallb/pull/1237) is merged.